src/middleware/github-server-app-middleware.test.ts

```diff
@@ -59,24 +57,25 @@ describe("github-server-app-middleware", () => {
 			gitHubClientSecret: "myghsecret",
 			webhookSecret: "mywebhooksecret",
 			privateKey: "myprivatekey",
-			installationId: JIRA_INSTALLATION_ID
+			installationId: JIRA_INSTALLATION_ID,
+			decrypt: async (s: any) => s
 		};
 
 		installation = {
-			jiraHost: "https://testatlassian.com",
+			jiraHost,
```

Comment:
We don't need to manually set the jiraHost. It's available in the global variables defined in setup.